content/en/seachat/seachat-manual/03-add-knowledge/02-spreadsheet-upload.md

```diff
@@ -11,6 +11,10 @@ aliases:
   - /en/seachat/seachat-manual/03-add-knowledge/02-spreadsheet-upload
 ---
 
+> 🧭 **File Size Rule**
+> 
+> According to your subscription plan, you can upload files up to a certain size. If you exceed the file upload limitation, you will receive an error message. Please refer to the [File Size Rule](https://wiki.seasalt.ai/seachat/seachat-manual/03-add-knowledge/file-size-rule/) for more information.
```

Comment:
Your file size limit for each uploaded document varies depending on your subscription plan. If you exceed the file upload limitation, you will receive an error message. Please reduce the size of your file before uploading again. Please refer to the [File Size Rule](https://wiki.seasalt.ai/seachat/seachat-manual/03-add-knowledge/file-size-rule/) for more information.

Reply:
change to all section